build.gradle

```diff
@@ -262,7 +262,7 @@ subprojects {
 
   tasks.withType(SpotBugsTask) {
     effort = 'max'
-    reportLevel = 'low'
+    reportLevel = 'high'
```

Comment:
Hmm... maybe I don't understand this config properly but I think we would still want the report to include all issues, even if not all issues are allowed to fail the build...

This is the behavior we had on the internal Jenkins, and probably the one we wish to keep... Why do we need to change it here? Might be good to check if we pass different arguments to the build on Jenkins to achieve the desired outcome.

Reply:
I researched this a bit and turned out there's no way of doing this, which is unfortunate. 

I spent some time to fix most of them to keep the reportLevel as `low` and still make it report the bugs